core/src/Root.ts

```diff
@@ -553,24 +564,26 @@ export class RootAPI implements Types.RootAPI {
         let mover: HTMLElement | undefined;
         let moverOptions: Types.MoverOptions | undefined;
         let isGroupperFirst: boolean | undefined;
+        let isRtl = false;
 
-        for (let e: (Node | null) = element; e; e = e.parentElement) {
-            const tabsterOnElement = getTabsterOnElement(tabster, e);
+        let curElement: (Node | null) = element;
+        while (curElement && (!root || options.checkRtl )) {
+            const tabsterOnElement = getTabsterOnElement(tabster, curElement);
 
             if (!tabsterOnElement) {
+                curElement = curElement.parentElement;
                 continue;
             }
 
             if (!groupper && tabsterOnElement.groupper) {
                 groupper = tabsterOnElement.groupper;
             }
 
-            const cfk = tabsterOnElement.focusable?.mover;
```

Comment:
There were two points to have this const: it minifies almost twice as better and (yeah, pretty little but still) is saves from the secondary lookup.

Reply:
alright, can we have a better name than `cfk` ? like `moverElement` ?

Reply:
I believe cfk was initially standing for currentFocusKey (or something like that). I suppose moverOnElement (or moe :)) would be consistent with tabsterOnElement :).

Reply:
moverOnElement it is :)